app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  attr_reader :keyword
+
+  def perform(keyword_id)
+    @keyword = Keyword.includes(:source).find(keyword_id)
+    search_result = search_service
```

Comment:
```suggestion
    search_result = call_search_service
```

Or you can propose a better name. Because let think `search_result = search_service` - it is a bit confusing. 🎨